week2/homework/src/list.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+const desc = 'LIST: shows the toDoList';
+const toDos = fs.readFileSync('toDoList.txt', 'utf8');
```

Comment:
What would happen if there is no such file?

Reply:
I decided to create an empty txt when the js file called (node .)